src/neural/cuda/winograd_helper.h

```diff
@@ -0,0 +1,387 @@
+/*
```

Comment:
I'm not sure but this almost feels more like a .inc than a .h - maybe?

Reply:
Exactly. I had to put the code in that file and include from two source files because different template instances of the same functions need to be compiled with different SM versions. Renamed to .inc